### PR TITLE
Removed consul reload from maybe-upgrade script

### PR DESCRIPTION
**Pull request by amar266**

### CODE REVIEW

**Comment by amar266** [comment]
please test this

**Comment by anshprat** [comment]
please test this

**Comment by amar266** [comment]
please test this

File: templates/consul-alerts.erb [hunk 1]
```diff
@@ -2,9 +2,14 @@ description "Consul Alert"
 start on (local-filesystems and net-device-up IFACE!=lo)
 stop on runlevel [06]
 
-env CONSUL_ALERTS=/usr/bin/consul-alerts
 
-exec sudo -u consul -g consul $CONSUL_ALERTS start --watch-events --watch-checks --consul-dc <%= @consul_discovery_token %>
+script
+  . /etc/environment
+  export http_proxy
```
Comment (hkumarmk) [inline templates/consul-alerts.erb]: isnt it "export $http_proxy" rather than "export http_proxy"?

**Comment by hkumarmk** [comment]
+1 for removing consul reload from maybe_upgrade.sh. I dont see it make sense to reload consul without any reason (like consul failure/config change etc)

**Comment by hkumarmk** [comment]
please test this

File: templates/consul-alerts.erb (hunk 1 above)
Comment (amar266) [inline templates/consul-alerts.erb]: it's fine , after running . /etc/environment , the syntax is export http_proxy  in upstart scripts as per docs

**Comment by bodepd** [comment]
please test this

**Comment by bodepd** [comment]
I'd like to see this pass several times before we merge it.

**Comment by bodepd** [comment]
please test this

**Comment by amar266** [comment]
please test this

**Comment by amar266** [comment]
please test this

**Comment by bodepd** [comment]
Getting comments here as to these failures would be great.

**Comment by amar266** [comment]
After last green status all red status was due to error "requests.exceptions.ConnectionError: ('Connection aborted.', BadStatusLine("''",))" . I have checked those last two builds, inside build there was no error reported by jorc and all hosts were in member list.

**Comment by amar266** [comment]
please test this

**Comment by amar266** [comment]
this time it failed at the time of apply_resources apply was running with error  raise ConnectionError(err, request=request)
requests.exceptions.ConnectionError: ('Connection aborted.', BadStatusLine("''",))
Build step 'Execute shell' marked build as failure

**Comment by amar266** [comment]
please test this

**Comment by amar266** [comment]
please test this

File: templates/consul-alerts.erb [hunk 2]
```diff
@@ -2,9 +2,14 @@ description "Consul Alert"
 start on (local-filesystems and net-device-up IFACE!=lo)
 stop on runlevel [06]
 
-env CONSUL_ALERTS=/usr/bin/consul-alerts
```
Comment (bodepd) [inline templates/consul-alerts.erb]: why is this entirely unrelated change here and not even mentioned in the commit message?

Comment (amar266) [inline templates/consul-alerts.erb]: The consul-alerts was not able to send data to slack because the env for proxy was not available in production when the service was started there by puppet, so I made changes in upstart script

**Comment by amar266** [comment]
please test this

**Comment by amar266** [comment]
Please test this

**Comment by amar266** [comment]
please test this

**Comment by amar266** [comment]
please test this

**Comment by amar266** [comment]
It failed earlier due to ssh time out.

**Comment by bodepd** [comment]
I think this is a sufficient amount of testing, but I'd like to hold off deploying if for a few days given the current large amount of code waiting to be merged into prod. Once our large amount of changes gets merged into prod, I'd like to see this as part of the next prod update.

**Comment by anshprat** [comment]
@bodepd this looks good to go now?
